tripgo.swagger.yaml

```diff
@@ -170,6 +170,312 @@ paths:
           schema:
             $ref: '#/definitions/RegionInfoResponse'
 
+  /info/operator.json:
+    post:
+      tags:
+        - Configuration
+      summary: List of TSPs for a particular region.
+      description:
+        Retrieves detailed information about covered transport service providers for a specified region.
+      parameters:
+        - name: input
+          in: body
+          schema:
+            type: object
+            description: Use `modes` to filter by tranport modes. Only results for the given modes will be retuned. When ´full´ paramater is false, it will retrieve a list of operator IDs for the given region and ´modes´ and ´onlyRealTime´ if any. You can also retrieve information for a list of particular operators using ´operatorsID´, providing their IDs as they were given before as a list member in this very same endpoint. The use of filters like ´modes´, ´onlyRealTime´ and/or ´operatorsID´ could lead to a empty response. In order to prevent this, make sure such combinations are posible, using previously fetched information.
+            properties:
+              region:
+                type: string
+                description: Region name/code from `regions.json`.
+              modes:
+                type: array
+                items:
+                  type: string
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              operatorsIDs:
+                type: array
+                items:
+                  type: string
+                description: Operators ID to retrieve information from. In case of any missmatch, it'll return error response.
+              onlyRealTime:
+                type: boolean
+                description: Boolean to filter only operators with Real Time support. When true, returned operator list will contain only TSP with Real Time service. False by default.
+              full:
+                type: boolean
+                description: Boolean if result should include full information for operator/s or just a list with all their IDs ignoring ´mode´ array in response. False by default.
```

Comment:
There is no deep thought on this. My original intention was since you don't know how heavy (large) the response is going to be, you will get a list with their names/ids and then you can filter the ones you desire and hit it again, this time with full on. But I can change that to true by default. It's not a issue at all.

I didn't know about  `default: false`. I'm learning as I go, and missed that one.